sql/updates/world/2015_12_10_14_world.sql

```diff
@@ -0,0 +1,9 @@
+--
+-- Landmine Knockback Achievement (29397,54355,54402,57099)
+DELETE FROM `spell_linked_spell` WHERE `spell_trigger` = 54355;
+INSERT INTO `spell_linked_spell` (`spell_trigger`, `spell_effect`, `comment`) VALUES
+(54355, 54402, 'Landmine Knockback Achievement');
+
```

Comment:
OK, but isn't entry 5258 a Cataclysm achievement? (I will change this anyway, just curious.)